fabtests/multinode/src/core.c

```diff
@@ -221,10 +237,11 @@ int multi_msg_recv()
 		offset = state.recvs_posted % opts.window_size;
 		assert(rx_ctx_arr[offset].state == OP_DONE);
 
+		remote_fi_addr = pm_job.fi_addrs[state.cur_source];
 		ret = ft_post_rx_buf(ep, opts.transfer_size,
 				     &rx_ctx_arr[offset].context,
 				     rx_ctx_arr[offset].buf,
-				     rx_ctx_arr[offset].desc, 0);
+				     rx_ctx_arr[offset].desc, 1);
```

Comment:
This change shouldn't be needed.  A tag of 0 is fine.

Reply:
This has to be 1 because of https://github.com/ofiwg/libfabric/blob/main/fabtests/common/shared.c#L2233

Reply:
thanks!